<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyProgress</class>
 <widget class="QDialog" name="CopyProgress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>119</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>188</width>
    <height>119</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>188</width>
    <height>119</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Копирование</string>
  </property>
  <widget class="QLabel" name="textLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Подождите, идет копирование...</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
